js-extensions/index.js

```diff
@@ -1,9 +1,20 @@
 // Provide an ExtensionStore & ExtensionRenderer react component
-exports.store = require('./dist/ExtensionStore.js').instance;
+const ExtensionStore = require('./dist/ExtensionStore.js').default;
+exports.store = new ExtensionStore();
 
-exports.Renderer = require('./dist/ExtensionRenderer.js').ExtensionRenderer;
+exports.Renderer = require('./dist/ExtensionRenderer.js').default;
 
-exports.classMetadataStore = require('./dist/ClassMetadataStore.js').instance;
+//in lieu of DI
+const ResourceLoadTracker = require('./dist/ResourceLoadTracker.js').default;
+const resourceLoadTracker = new ResourceLoadTracker();
+exports.store.ResourceLoadTracker = resourceLoadTracker;
```

Comment:
And why `exports.store.ResourceLoadTracker` as opposed to just `exports.ResourceLoadTracker` ? Just curious. Maybe add a comment as to why it hangs there.